src/Data/Nat/Primality.agda

```diff
@@ -141,3 +355,4 @@ private
   -- Example: 6 is composite
   6-is-composite : Composite 6
   6-is-composite = from-yes (composite? 6)
```

Comment:
Probably need to expand the comment in this section, showing how you can use `Decidable` to produce proofs of primality automatically? As opposed to above where you do it manually?

Reply:
Those examples were inherited from the previous version... but yes, perhaps these examples better belong in a `README.Primality`?

Reply:
Added some text around the examples, towards eventually (but not now!) writing a `README.Primality`...